example/consumer.js

```diff
@@ -43,12 +47,12 @@ const steveoConfig = {
   const steveo = Steveo(config, logger)();
 
   steveo.events.on('runner_failure', (topic, ex) => {
-    logger.debug('Failed to call subscribe', topic, ex);
+    logger.info('Failed to call subscribe', topic, ex);
   });
 
   // subscribe Call for first task
   const subscribe = async payload => {
-    logger.debug('Payload from producer', payload);
+    logger.info('Payload from producer', payload);
```

Comment:
@arshu16 switch back to debug and pass in a higher logger level in the constructor to see these messages?